src/nyc_trees/apps/survey/models.py

```diff
@@ -55,6 +60,59 @@ class Tree(NycModel, models.Model):
     # We do not anticipate deleting a Species, but we definitely
     # should not allow it to be deleted if there is a related Tree
     species = models.ForeignKey(Species, null=True, on_delete=models.PROTECT)
+    distance_to_tree = models.FloatField()
+    distance_to_end = models.FloatField(null=True)
+    circumference = models.FloatField()
+    status = models.CharField(max_length=15, choices=(
+        ('Alive', 'Tree is alive'),
+        ('Dead', 'Tree is dead'),
+        ('Stump', 'Stump [<24"]'),
+    ))
+    species_certainty = models.CharField(max_length=15, choices=(
+        ('Yes', 'Yes'),
+        ('No', 'No'),
+        ('Maybe', 'Maybe'),
+    ))
+    curb_location = models.CharField(max_length=25, choices=(
+        ('OnCurb', 'Tree bed is along the curb'),
+        ('OffsetFromCurb', 'Tree bed is offset from the curb'),
+    ))
+    health = models.CharField(max_length=15, choices=(
+        ('Good', 'Good'),
+        ('Fair', 'Fair'),
+        ('Poor', 'Poor'),
+    ))
+    stewardship = models.CharField(max_length=15, choices=(
+        ('None', 'Zero'),
+        ('1or2', 'One to two'),
+        ('3or4', 'Three to four'),
+        ('4orMore', 'More than four'),
+    ))
+    guards = models.CharField(max_length=15, choices=(
+        ('None', 'No guard installed'),
+        ('Helpful', 'Helpful guard installed'),
+        ('Harmful', 'Harmful guard installed'),
+        ('Unsure', 'Unsure if guard is helpful or harmful'),
+    ))
+    sidewalk_damage = models.CharField(max_length=15, choices=(
+        ('NoDamage', 'No damage is seen'),
+        ('Damage', 'Cracks or raised sidewalk seen'),
+    ))
+    problems = models.CharField(max_length=130)
+    # TODO: Multiple problems may be selected:
+    #     ('None', 'No problems'),
+    # "Root problems"
+    #     ('Stones', 'Sidewalk or stones'),
+    #     ('MetalGrates', 'Metal grates'),
+    #     ('RootOther', 'Other'),
+    # "Trunk problems"
+    #     ('WiresRope', 'Wires or rope'),
+    #     ('TrunkLights', 'Lights'),
+    #     ('TrunkOther', 'Other'),
+    # "Branch problems"
+    #     ('BranchLights', 'Lights'),
+    #     ('Sneakers', 'Sneakers'),
+    #     ('BranchOther', 'Other'),
```

Comment:
Consider adding a clean method that ensures the `problems` field is a comma separated list containing at most 1 of each of these choice values.

We don't want incorrect fields getting in here via the Admin site.